core/src/main/java/hudson/model/User.java

```diff
@@ -655,7 +655,7 @@ private boolean relatedTo(@Nonnull AbstractBuild<?,?> b) {
     @SuppressWarnings("unchecked")
     @WithBridgeMethods(List.class)
     public @Nonnull RunList getBuilds() {
-        return RunList.fromJobs(Jenkins.getInstance().allItems(Job.class)).filter(new Predicate<Run<?,?>>() {
+        return RunList.fromJobs((Iterable)Jenkins.getInstance().allItems(Job.class)).filter(new Predicate<Run<?,?>>() {
```

Comment:
Might suffice to just replace `Run<?,?>` with `Run` (untested). Or to add a qualifier to `filter` like

```java
return RunList.fromJobs(Jenkins.getInstance().allItems(Job.class)).<Something<Here>>filter(new Predicate<Run<?,?>>() {
```